app/assets/javascripts/manage_submissions.js

```diff
@@ -1,3 +1,17 @@
+var submission_info = {}
+let tweaks = [];
```

Comment:
nit: Instead of defining tweaks up here as a global var, can we define it on line 95, then pass tweaks into updateTweaks as a parameter

Reply:
updateTweaks is also called in submitNewAnotation in annotations_popup.js, so it would be out of scope if we moved it to line 95 :/